### ISSUE TITLE
OnlyOffice DesktopEditor build on Windows 10 with MSVC 2015

### ISSUE THREAD

**Issue by jinjingyu** [ONLYOFFICE/DesktopEditors#118]
Hello,
Nice to meet you all, my name is xiaojin and i am developer from china.

I got the lastest DesktopEditor source code from github.com.
I build OnlyOffice DestopEditor with MSVC 2015 64bit on Windows 10 and successed
And I run Office.
At the first start of the software the main window appears correctly but  whether I try to create a new document or open an existing one, DesktopEditors takes many minutes and show error message like "An error has occurred whiel opening the file".

Thanks for your attention.

Best regards

**Comment by ShockwaveNN** [comment]
@jinjingyu can you describe your build process?

**Comment by ShockwaveNN** [comment]
@jinjingyu Can I ask how did you find this instruction? It should be hidden.
This is Work In Progress variant and I think shouldn't work, currently we have no fully working instruction, sorry for your time.
We added notice at start of instruction to clearly state this

**Comment by ShockwaveNN** [comment]
@jinjingyu https://helpcenter.onlyoffice.com/desktop/documents/windows/compile-desktop-windows.aspx
![image](https://user-images.githubusercontent.com/668524/41597131-3daa9e14-73d5-11e8-970e-4e8ddcea6db2.png)
My question about how did you find this page is still actual :)

**Comment by jinjingyu** [comment]
I found it like following 
1. I search key "how compile OnlyOffice DesktopEditors for windows" in www. goolge.com
![image](https://user-images.githubusercontent.com/40283634/41597736-dd7f47f2-7400-11e8-9bd5-396c5b17b873.png)
2. I replace ubuntu to windows
the end.
I interest office program. I used to find Office Open Source. At this time, I interested your project very much. I hope I am working with you. 
thanks
